themes/bootstrap3/js/channels.js

```diff
@@ -13,7 +13,7 @@ VuFind.register('channels', function Channels() {
     }
     var $cont = $(
       '<div class="dropdown">' +
-        '<button class="btn btn-link" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">' +
+        '<button class="btn btn-link" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true" aria-label="' + VuFind.translate('toggle_dropdown') + '">' +
```

Comment:
Did you test that this translates correctly? I'm pretty sure it won't work unless the string is registered in the jsTranslations helper (typically done either in [js-translations.phtml](https://github.com/vufind-org/vufind/blob/dev/themes/bootstrap3/templates/layout/js-translations.phtml) or in feature-specific code for more narrowly-scoped/rarely-used translations.

Reply:
Thanks @demiankatz ! 
I missed that